docs/wiki/GettingStarted.md

```diff
@@ -212,7 +212,7 @@ Note: There are default tokens that can be enabled on any resource that leverage
 - `<<tenantId>>`: Will point to the Azure Tenant ID.
 - `<<deploymentSpId>>`: Will point to the Service Principal ID used for deployments.
 - `<<platformKeyVault>>`: Will point to the Platform Azure Key Vault (Optional if enabled as per guidelines)
-- `<<resourceGroupName>>`: Will point to the Azure Resource Group where the resources are being deployed to. (This isn't defined in the secrets section but is injected at runtime)
+- `validation-rg`: Will point to the Azure Resource Group where the resources are being deployed to. (This isn't defined in the secrets section but is injected at runtime)
```

Comment:
I think this happened due to search & replace

Reply:
good pickup. fixed